### PR TITLE
fix(select): fix caret icon to show disabled styling

### PR DESCRIPTION
**Pull request by KrisKnabel**
Caret icon is not showing disabled, this change fixes the caret icon to display properly when
disabled.  Pointed out by the Cedar Designers and they requested the fix.

### CODE REVIEW

File: src/components/select/styles/vars/CdrSelect.vars.scss [hunk 1]
```diff
@@ -167,3 +167,19 @@ $cdr-select-helper-text-pt: $cdr-space-quarter-x;
   padding-left: $cdr-select-helper-text-pl;
   padding-top: $cdr-select-helper-text-pt;
 }
+
+/* caret */
+@mixin cdr-select-base-caret-mixin() {
+  fill: $cdr-color-icon-primary-lightmode;
```
Comment (cowills) [inline src/components/select/styles/vars/CdrSelect.vars.scss]: you need to create variables for all these properties if you're going to map these like this. It's fine to just leave this CSS un-mapped instead.

**Review by lancetharper — DISMISSED** [review]

**Review by cowills — CHANGES_REQUESTED** [review]

File: src/components/select/styles/vars/CdrSelect.vars.scss (hunk 1 above)
Comment (KrisKnabel) [reply src/components/select/styles/vars/CdrSelect.vars.scss]: When you say variables, are you referring to Cedar Component Variables?

Comment (cowills) [reply src/components/select/styles/vars/CdrSelect.vars.scss]: the `vars.scss` files are the component variables. if you aren't going to map each of these values to a variable then you should just keep these changes in the main scss file for select

Comment (KrisKnabel) [reply src/components/select/styles/vars/CdrSelect.vars.scss]: I see what you mean now.  Sorry for the confusion.  I was looking at the code above the comment, not below, and wondering if you were referring to running `npm run build:variables`.

Comment (KrisKnabel) [reply src/components/select/styles/vars/CdrSelect.vars.scss]: Updated the select caret style properties to map to SCSS variables.

**Comment by KrisKnabel** [comment]
Added Unit Test to cover Cedar Select Caret Disabled state.

**Comment by KrisKnabel** [comment]
Closing as PR 696 resolves the need for this.

**Pull request closed without merging**